### PR TITLE
Fix for permission and DB issues with /topic-groups

### PR DESCRIPTION
**Pull request by dodalovicgran**
Works locally :thinking: 
image

### CODE REVIEW

File: src/main/resources/application-local.properties [hunk 1]
```diff
@@ -26,8 +26,8 @@ spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MariaDB103Dialect
 
 # Liquibase
 spring.liquibase.change-log=classpath:db/changelog/consultingtypeservice-local-master.xml
-spring.liquibase.user=liquibase
```
Comment (tkuzynow) [inline src/main/resources/application-local.properties]: please revert this

File: src/main/java/de/caritas/cob/consultingtypeservice/config/SecurityConfig.java [hunk 2]
```diff
@@ -76,7 +76,7 @@ protected void configure(final HttpSecurity http) throws Exception {
         .requestMatchers(new AntPathRequestMatcher("/topic/*"))
         .authenticated()
         .requestMatchers(new AntPathRequestMatcher("/topic-groups"))
-        .authenticated()
+        .permitAll()
```
Comment (tkuzynow) [inline src/main/java/de/caritas/cob/consultingtypeservice/config/SecurityConfig.java:80]: it should have same permissions as topic IMO, permit all? are you not planning to add some management operations there?

Comment (dodalovicgran) [reply src/main/java/de/caritas/cob/consultingtypeservice/config/SecurityConfig.java:80]: Actually, I can't tell much here, but I was told by @web-mi to do so. :shrug:

**Review by tkuzynow — APPROVED** [review]

File: src/main/java/de/caritas/cob/consultingtypeservice/config/SecurityConfig.java (hunk 2 above)
Comment (web-mi) [reply src/main/java/de/caritas/cob/consultingtypeservice/config/SecurityConfig.java:80]: Currently we will not integrate the topic groups inside the admin console.

**Pull request merged**